apps/synapse-portal-framework/src/components/NavUserLink.tsx

```diff
@@ -20,12 +20,6 @@ function NavUserLink({ userProfile }: NavUserLinkProps) {
         link="javascript:void(0)"
       />
       <ExpandMoreIcon className="user-expand" />
-      <div className="mb-user-extra">
```

Comment:
Move outside NavUserLink, it's no longer all pointing to a single URL (to the user profile)